frontend/packages/dev-console/src/components/monitoring/overview/MonitoringOverview.tsx

```diff
@@ -78,6 +79,15 @@ const MonitoringOverview: React.FC<MonitoringOverviewProps> = ({ resource, event
             Metrics
           </AccordionToggle>
           <AccordionContent id="metrics-content" isHidden={!expanded.includes('metrics')}>
+            <h5 className="odc-monitoring-overview__view-monitoring-dashboard text-right">
```

Comment:
Why an `h5` ?
Should use pf4 utilities like `pf-u-text-align-right` if available instead of bootstrap.

Reply:
removed bootstrap ```text-right``` class and defined it in ```odc-monitoring-overview__view-monitoring-dashboard``` this class. Didn't find any pf4 utility for this.

Reply:
hmm must be because we aren't importing those utilities. oh well